e2e/rdr-e2e.sh

```diff
@@ -1,4 +1,11 @@
-#!/bin/bash
+#!/bin/sh
 
-echo "Test ran successfully"
-exit 0
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Running tests..."
+
+cd ./e2e/
```

Comment:
@par97 there is a shellcheck linter failure here: https://github.com/RamenDR/ramen/actions/runs/8835433743/job/24299496088?pr=1352#step:5:19 that would need to be fixed accordingly.

Reply:
fixed shellcheck issue